jwst/jump/tests/test_detect_jumps.py

```diff
@@ -12,7 +12,7 @@ def test_nocrs_noflux(setup_inputs):
     all pixel values are zero. So slope should be zero
     """
     model1, gdq, rnModel, pixdq, err, gain = setup_inputs(ngroups=5)
-    out_model = detect_jumps(model1, gain, rnModel, 4.0,  1, 200, 4, True)
+    out_model = detect_jumps(model1, gain, rnModel, 4.0,  'one', 200, 4, True)
```

Comment:
If max_cores is set to 'one', it will set num_slices to be 1. Else, it will look at the value of max_cores for a match and if it doesn't match set num_slices to 1. So, it's a little faster when max_cores is set to 'one' and logically cleaner than just letting it cascade through the ifs and end up with num_slices = 1.